src/main/kotlin/minesweeper/domain/board/NearbyMineCounter.kt

```diff
@@ -0,0 +1,52 @@
+package minesweeper.domain.board
+
+import minesweeper.domain.cell.Cell
+import minesweeper.domain.cell.Empty
+import minesweeper.domain.common.PositiveInt
+import minesweeper.domain.common.compareTo
+import minesweeper.domain.common.plus
+import minesweeper.domain.common.times
+
+class NearbyMineCounter {
+
+    companion object {
+        fun count(mineBoard: MineBoard) {
+            mineBoard.cells.mineIndices.forEach { index ->
+                accNearbyMines(mineBoard, mineBoard.cells[index])
+            }
+        }
+
+        private fun accNearbyMines(mineBoard: MineBoard, mineCell: Cell) {
```

Comment:
```suggestion
        private fun accNearbyMines(cells:Cells) {
```
NearbyDirection와 cells만으로도 해당 cell에 인접한 MineCell의 개수를 구할 수 있어 보여요. 개선해 볼까요?